### PR TITLE
Addfunds ui

### PR DESCRIPTION
**Pull request by aekeus**
Submitter Checklist:

- [X] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [X] Ran `git rebase -i` to squash commits (if needed).
- [X] Tagged reviewers and labelled the pull request [as needed](https://github.com/brave/browser-laptop/wiki/Pull-request-process).

Test Plan:

  Per QA - @srirambv 

Related to: #8694

### CODE REVIEW

File: app/ledger.js [hunk 1]
```diff
@@ -1692,6 +1714,50 @@ var callback = (err, result, delayTime) => {
...
+  }
+  request.request(options, (err, response, body) => {
+    if (err) {
+      return callback(null)
+    }
+    var customer = response.statusCode === 200 ? JSON.parse(body) : null
+    if (customer && process.env.LEDGER_LAST_PURCHASE_DAYS_AGO) {
+      customer.lastChargeTimestamp = parseInt(moment().subtract(parseInt(process.env.LEDGER_LAST_PURCHASE_DAYS_AGO), 'days').format('x'))
+    }
+    if (customer && clientOptions.debugP) {
```
Comment (bsclifton) [inline app/ledger.js]: Should we consider logging non-200 responses? ex: If we're constantly getting an error, would be nice to know if it's a 500 error (server) or a 401 (access)... along with details about the error 😄  (presumably in body)

File: app/renderer/components/preferences/payment/advancedSettings.js [hunk 2]
```diff
@@ -79,7 +80,17 @@ class AdvancedSettingsContent extends ImmutableComponent {
             className={css(styles.listItem, commonStyles.noMarginBottom)}
             switchClassName={css(styles.checkboxSwitch)}
             labelClassName={css(commonStyles.noMarginBottom)}
-          />
+      />
+
+          {
+            !this.props.ledgerData.getIn(['customer', 'transactions']) ? null
+              : <div className={cx({[css(styles.refundLinkContainer)]: true})}>
```
Comment (luixxiul) [inline app/renderer/components/preferences/payment/advancedSettings.js]: `<div className={css(styles.refundLinkContainer)}>`

File: app/renderer/components/preferences/payment/advancedSettings.js [hunk 3]
```diff
@@ -79,7 +80,17 @@ class AdvancedSettingsContent extends ImmutableComponent {
...
+                <a href={this.props.ledgerData.get('refundURL')}
+                  target='_blank'
+                  data-l10n-id='getARefund'
+                  className={cx({[css(styles.refundLink)]: true})} />
```
Comment (luixxiul) [inline app/renderer/components/preferences/payment/advancedSettings.js]: also in BEM style: `<div className={css(styles.refundLinkContainer__refundLink)}>`

File: app/renderer/components/preferences/payment/enabledContent.js [hunk 4]
```diff
@@ -32,6 +32,23 @@ const aboutActions = require('../../../../../js/about/aboutActions')
...
 // TODO: support non-USD currency
 class EnabledContent extends ImmutableComponent {
+  customerButton () {
+    const ledgerData = this.props.ledgerData.toJSON()
+    const buttonText = 'manageAutorenewal'
+    const URL = ledgerData.customerURL
+    return <div
+      className={cx({
+        [css(styles.autoRenewalLinkContainer)]: true
+      })}>
```
Comment (luixxiul) [inline app/renderer/components/preferences/payment/enabledContent.js]: `<div className={css(styles.autoRenewalLinkContainer)}>`

File: app/renderer/components/preferences/payment/enabledContent.js [hunk 5]
```diff
@@ -32,6 +32,23 @@ const aboutActions = require('../../../../../js/about/aboutActions')
...
+      <a href={URL}
+        target='_blank'
+        data-l10n-id={buttonText}
+        className={cx({
+          [css(styles.autoRenewalLink)]: true
+        })} />
```
Comment (luixxiul) [inline app/renderer/components/preferences/payment/enabledContent.js]: `className={css(styles.autoRenewalLinkContainer_link) />`

File: app/renderer/components/preferences/payment/advancedSettings.js [hunk 6]
```diff
@@ -139,6 +150,13 @@ const styles = StyleSheet.create({
     marginTop: '2px',
     paddingTop: 0,
     paddingBottom: 0
+  },
+  refundLinkContainer: {
+    marginTop: '8px'
+  },
+  refundLink: {
```
Comment (luixxiul) [inline app/renderer/components/preferences/payment/advancedSettings.js]: `refundLinkContainer__link`

File: app/renderer/components/preferences/payment/enabledContent.js [hunk 7]
```diff
@@ -323,6 +341,15 @@ const styles = StyleSheet.create({
   walletStatus: {
     marginTop: '15px',
     lineHeight: 1.5
+  },
+
+  autoRenewalLinkContainer: {
+    marginTop: '8px'
+  },
+
+  autoRenewalLink: {
```
Comment (luixxiul) [inline app/renderer/components/preferences/payment/enabledContent.js]: `autoRenewalLinkContainer__link`

**Comment by srirambv** [comment]
I do not see request refund option in the browser (0cdfc42).  Add funds button is still disabled but adding env variable able to open add funds page

**Comment by aekeus** [comment]
Deprecated. Closing until further notice.

**Comment by luixxiul** [comment]
@aekeus Is it OK to remove this PR from the milestone, or will you reopen the PR?

**Comment by aekeus** [comment]
Thanks @luixxiul. Removing from the milestone.

**Pull request closed without merging**